<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>905</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="DEV_LIST">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>301</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox:title {
subcontrol-origin: margin;
subcontrol-position: top center;
padding-left: 120px;
padding-right: 120px; 
padding-top: 5px;
padding-bottom: 5px;
border: 2px solid black;
border-radius: 1px;
background-color: rgb(0,0,0)}

QGroupBox {
border: 2px solid gray;
border-radius: 5px;
font: 10pt &quot;Arial Black&quot;;
color:rgb(255,255,255);
}
</string>
    </property>
    <property name="title">
     <string>Device List</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>281</width>
       <height>201</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:1px solid black;
border-radius: 5px;
font: 8pt &quot;Arial Black&quot;;
color:rgb(0,0,0);

</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="CTL_WIDGET">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>571</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox:title {
subcontrol-origin: margin;
subcontrol-position: top center;
padding-left: 250px;
padding-right: 250px; 
padding-top: 5px;
padding-bottom: 5px;
border: 2px solid black;
border-radius: 1px;
background-color: rgb(0,0,0)}

QGroupBox {
border: 2px solid gray;
border-radius: 5px;
font: 10pt &quot;Arial Black&quot;;
color:rgb(255,255,255);
}
</string>
    </property>
    <property name="title">
     <string>Controll Widget</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QTabWidget" name="CTL_TAB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>551</width>
       <height>201</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid black;
border-radius: 3px;
font: 8pt &quot;Arial&quot;;
color:rgb(0,0,0);

</string>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="POWER">
      <attribute name="title">
       <string>POWER</string>
      </attribute>
      <widget class="QLabel" name="DEV_NAME">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 0px solid;
font: 8pt &quot;Arial Black&quot;;
color:rgb(255,255,255);
background-color:rgb(0,0,0);
</string>
       </property>
       <property name="text">
        <string>DEV</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="FREQ">
      <attribute name="title">
       <string>FREQ</string>
      </attribute>
      <widget class="QLabel" name="DEV_NAME2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 0px solid;
font: 8pt &quot;Arial Black&quot;;
color:rgb(255,255,255);
background-color:rgb(0,0,0);
</string>
       </property>
       <property name="text">
        <string>DEV</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="PHASE">
      <attribute name="title">
       <string>PHASE</string>
      </attribute>
      <widget class="QLabel" name="DEV_NAME3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 0px solid;
font: 8pt &quot;Arial Black&quot;;
color:rgb(255,255,255);
background-color:rgb(0,0,0);
</string>
       </property>
       <property name="text">
        <string>DEV</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="STATUS_DIS">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>881</width>
      <height>311</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox:title {
subcontrol-origin: margin;
subcontrol-position: top center;
padding-left: 500px;
padding-right: 500px; 
padding-top: 5px;
padding-bottom: 5px;
border: 2px solid black;
border-radius: 1px;
background-color: rgb(0,0,0)}

QGroupBox {
border: 2px solid gray;
border-radius: 5px;
font: 10pt &quot;Arial Black&quot;;
color:rgb(255,255,255);
}</string>
    </property>
    <property name="title">
     <string>Current Status Display</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
